XYPlot/Graph tile toolbar refactor

bgoldowsky opened the pull request:
Refactors the toolbar to use the new toolbar system, satisfying PT-186143196.

This also automatically makes the toolbar configurable, so it satisfies PT-186318185.

bgoldowsky commented:
Note, there are some updates that will be desirable or required for this after #2016 is merged: putting defaults in app-config.json, moving the tooltip definition, using new Cypress method to locate toolbar buttons.

scytacki reviewed (changes requested):
I'd recommend updating this PR so it is "based" on the branch of the other PR: https://github.com/concord-consortium/collaborative-learning/pull/2016

This way you won't need a 3rd PR to fixing things up after 2016 is merged.

Other than that it looks great. It is nice how small this change is and it is mostly removing code.

bgoldowsky commented:
Merged in the table-toolbar-refactor branch and changed the base branch of this PR to point there.

I also did a little cleanup: since now all tiles in the new system use AppConfig to register defaults, I removed the redundant method of registering the default button list.  Also updated some comments and deleted a couple of old text-toolbar files that got missed when I did that refactor.

scytacki reviewed (approved):
looks great 👍

tejal-shah reviewed (approved):
LGTM 👍
